code/cryptography/src/baconian_cipher/baconian.cpp

```diff
@@ -0,0 +1,81 @@
+#include <map>
+#include <string>
+
+#define VALUE_LENGTH 5 // The length of a value in a key value pair for a baconian cipher
```

Comment:
Why not a constant variable. If yes, then I would move it inside main since global variables are bad.